### ISSUE TITLE
DetailsList - Grouped shows 'ARIA attribute cannot be used, add a role attribute or use a different element: aria-label' failed instance if there is no item in group 

### ISSUE THREAD

**Issue by SummerCat200** [microsoft/fluentui#19755]
### Environment Information

- **Package version(s)**: 8.33.0
- **Browser and OS versions**: Edge

### Please provide a reproduction of the issue in a codepen:
Run fastpass for accessibility insights for web extension on [DetailsList - Grouped](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped) page, it shows 'ARIA attribute cannot be used, add a role attribute or use a different element: aria-label' failed instance for Color green group

#### Actual behavior:

![image](https://user-images.githubusercontent.com/52942564/132902077-efa47570-743f-4c13-b45b-130a9226957b.png)

**Comment by SummerCat200** [comment]
any updates

**Comment by SummerCat200** [comment]
Hi @GeoffCoxMSFT, any updates on this issue? Is there any workaround for it or do you need more details?

**Comment by gouttierre** [comment]
@SummerCat200 - Thanks for filing this issue with us. Our team will have a look but to set expectations, we’re likely not going to get to it immediately due to the fact we’re currently focused on larger coherence work items.

@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by SummerCat200** [comment]
Thanks @gouttierre, is there any workaround for it?
